src/convertToObject.js

```diff
@@ -3,16 +3,40 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes styles string
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const keyValueArr = removeSpecificCharacters(sourceString).split(';');
+  const resultObj = {};
+
+  keyValueArr.forEach(keyValue => {
```

Comment:
You can try to rewrite with the array method, which can return an object. 
In this case resultObj on the 16 line is not needed anymore)